File: 프로그래머스/lv1/12901. 2016년/2016년.cs
using System;
public class Solution{
public  string solution(int a, int b)
        {
            string answer = "";
            string[] dayName = new string[] { "FRI", "SAT" ,"SUN", "MON", "TUE", "WED", "THU", }; 
            int days = 0;

            for (int i = 1; i < a ; i++)
            {
                if (i == 1 || i == 3 || i == 5 || i == 7 || i == 8 || i == 10 || i == 12)
                    days += 31;
                else if (i == 2)
                    days += 29;
                else
                    days += 30;
            }
            days += b - 1;
            answer = dayName[days % 7];
            return answer;
        }
}

File: 프로그래머스/1/12901. 2016년/2016년.cpp
#include <string>
#include <vector>

using namespace std;

string solution(int a, int b) {
    vector<int>day = {31,29,31,30,31,30,31,31,30,31,30,31};
    vector<string>week =  {"THU","FRI","SAT","SUN","MON","TUE","WED"};
    string answer = " ";
    int sum = 0;
    for(int i = 0; i < a-1; i++)
    {
        sum += day[i];
        
    }
    sum += b;
    
    //if(sum%7-4 > 0)
   // {
      answer = week[sum%7];
  //  }
  //  else
  //  {
  //    int num = sum%7-4;
  //    answer = week[6+num];
        
    //}
    return answer;
}